yarn.lock

```diff
@@ -830,6 +830,14 @@ axios@^0.18.0:
     follow-redirects "1.5.10"
     is-buffer "^2.0.2"
 
+axios@^0.27.2:
```

Comment:
is `axios` already used, directly or in dependencies?
To keep the code maintainable, it is best to avoid introduced "redundant" deps, and use the same approach for similar uses (e.g. what is used when connecting to other APIs)

Reply:
replaced axios with native node http module